ibis/backends/spark/compiler.py

```diff
@@ -24,6 +24,7 @@
 import math
 
 import ibis
+import ibis.backends.base_sql
```

Comment:
can you use like
`import ibis.backends.base_sql as bsql` or similar

Reply:
I personally find using the name of the module much easier to read. Currently we're using (sample from `impala/compiler.py`):
```
import ibis.common.exceptions as com
import ibis.expr.analysis as L 
import ibis.expr.datatypes as dt
import ibis.expr.operations as ops
import ibis.expr.types as ir
import ibis.sql.compiler as comp
```
And seems like we're making our life very difficult for no reason. I'd rather keep the original module name, I find the code much clearer than keep adding aliases for everything.